docs/user_v1/explanations/status.rst

```diff
@@ -184,6 +184,35 @@ Note that ``set_finished``, ``subscribe`` and ``clear_sub`` are gone; they are
 handled automatically, internally. See
 :class:`~ophyd.status.SubscriptionStatus` for additional options.
 
+StableSubscriptionStatus
+------------------------
+
+The :class:`~ophyd.status.StableSubscriptionStatus` is a Status object that is
+similar to the :class:`~ophyd.status.SubscriptionStatus` but is only marked
+finished based on an ophyd event remaining stable for some given time. For
+example, this could be used to ensure a temperature remains in a given range
+for a set amount of time:
```

Comment:
There's an interesting subtlety here. Reading the docs, I would like to know whether the timer starts from the creation of the object or from the first response from the device. Also whether it starts again when the callback returns `False` or only after a `False` then a `True`. See options 1-4 below:


![image](https://github.com/bluesky/ophyd/assets/29771545/ecf9de8d-dc6c-49c9-9f31-dba69ef64438)

Reply:
Thanks, the answer is 4. I think this is sort of implied but you're right it's not poperly stated, I'll fix the docs